econml/dml.py

```diff
@@ -335,28 +390,87 @@ class DMLCateEstimator(_DMLCateEstimatorBase):
 
     inference: string, inference method, or None
         Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`)
+        (or an instance of `BootstrapOptions`) and 'statsmodels' (or an instance of 'StatsModelsOptions`)
     """
 
+    # add statsmodels to parent's options
+    _inference_options = {**DMLCateEstimator._inference_options, 'statsmodels': StatsModelsOptions()}
+
+    class StatsModelsWrapper:
```

Comment:
This can now be pulled out, but we should figure out the right place for it - utlities?

Reply:
I think either utilities or sth like: statsmodel_utilities.py (since it is utilities related to extending/wrapping the statsmodel package). Similarly I expect we'll have xgb_utilities.py where we wrap xgboost models

Reply:
I've moved this to utilities in a pending update.